packages/forklift-console-plugin/src/modules/Plans/views/details/tabs/VirtualMachines/Plan/PlanVMActionsDropdown.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, Ref, useState } from 'react';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { Dropdown, DropdownList, MenuToggle, MenuToggleElement } from '@patternfly/react-core';
+import { EllipsisVIcon } from '@patternfly/react-icons';
+
+import { PlanVMsCellProps } from '../components';
+
+import { PlanVMActionsDropdownItems } from './PlanVMActionsDropdownItems';
+
+export interface PlanVMActionsDropdownProps extends PlanVMsCellProps {
+  isKebab?: boolean;
+}
+
+const PlanVMActionsDropdown: FC<PlanVMActionsDropdownProps> = ({ data, isKebab }) => {
```

Comment:
is it going to be generic? seem very specifc to me, can omit the isKebab is its going to be kebab